packages/framework/presence/README.md

```diff
@@ -92,39 +92,11 @@ Notifications API is mostly unimplemented. All messages are always broadcast eve
 
 Notifications are fundamentally unreliable at this time as there are no built-in acknowledgements nor retained state. To prevent most common loss of notifications, always check for connection before sending.
 
-
-### Experimental
-
-<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README:scripts=FALSE) -->
+<!-- AUTO-GENERATED-CONTENT:START (README_FOOTER) -->
 
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
-**IMPORTANT: This package is experimental.**
-**Its APIs may change without notice.**
-
-**Do not use in production scenarios.**
-
-## Using Fluid Framework libraries
```

Comment:
We probably want to keep these sections that now come from `<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_README_HEADER) -->`, though they may want to go higher in the file. That said, we can always make that change in a follow-up PR, so no need to block the PR on this :)

Reply:
Added the header section